v21.2/crdb-internal.md

```diff
@@ -5,128 +5,639 @@ toc: true
+`invalid_objects` | Contains information about invalid objects in your cluster.| ✗
+`jobs` | Contains information about [jobs](show-jobs.html) running on your cluster.| ✗
+`kv_node_liveness` | Contains information about [node liveness](cluster-setup-troubleshooting.html#node-liveness-issues).| ✗
+`kv_node_status` | Contains information about node status at the [key-value layer](architecture/storage-layer.html).| ✗
+`kv_store_status` | Contains information about the key-value store for your cluster.| ✗
+`leases` | Contains information about [leases](architecture/replication-layer.html#leases) in your cluster.| ✗
+`lost_descriptors_with_data` | Contains information about table descriptors that have been deleted but still have data left over in storage.| ✗
+`node_build_info` | Contains information about nodes in your cluster.| ✗
+`node_contention_events` | Contains information about contention on the gateway node of your cluster.| ✗
+`node_distsql_flows` | Contains information about the flows of the [DistSQL execution](architecture/sql-layer.html#distsql) scheduled on nodes in your cluster.| ✗
+`node_inflight_trace_spans` | Contains information about currently in-flight spans in the current node.| ✗
+`node_metrics` | Contains metrics for nodes in your cluster.| ✗
+`node_queries` | Contains information about queries running on nodes in your cluster.| ✗
+`node_runtime_info` | Contains runtime information about nodes in your cluster.| ✗
+`node_sessions` | Contains information about sessions to nodes in your cluster.| ✗
+`node_statement_statistics` | Contains statement statistics for nodes in your cluster.| ✗
+`node_transaction_statistics` | Contains transaction statistics for nodes in your cluster.| ✗
+`node_transactions` | Contains information about transactions for nodes in your cluster.| ✗
+`node_txn_stats` | Contains transaction statistics for nodes in your cluster.| ✗
+`partitions` | Contains information about [partitions](partitioning.html) in your cluster.| ✗
+`predefined_comments` | Contains predefined comments about your cluster.| ✗
+`ranges` | Contains information about ranges in your cluster.| ✗
+`ranges_no_leases` | Contains information about ranges in your cluster, without leases.| ✗
+`regions` | Contains information about [cluster regions](multiregion-overview.html#cluster-regions).| ✗
+`schema_changes` | Contains information about schema changes in your cluster.| ✗
+`session_trace` | Contains session trace information for your cluster.| ✗
+`session_variables` | Contains information about [session variables](set-vars.html) in your cluster.| ✗
+[`statement_statistics`](#statement_statistics) | Aggregates in-memory and persisted [statistics](ui-statements-page.html#statement-statistics) from `system.statement_statistics` within hourly time intervals based on UTC time, rounded down to the nearest hour. To reset the statistics call `SELECT crdb_internal.reset_sql_stats()`.| ✓
```

Comment:
@Azhng At @kevin-v-ngo's request, @maryliag  removed that setting [here](https://github.com/cockroachdb/cockroach/commit/18f2177e5adf6b9d640efa4bd9ae8b495e4bb2eb)  because there isn't good advice available yet on what to set it to.

Reply:
Ah right! Forgot about that 😅 . 

Everything :LGTM: